src/resources/js/v2/tickets-commerce.js

```diff
@@ -136,9 +146,40 @@ tribe.tickets.commerce = {};
 		// Bind container based events.
 		obj.bindCheckoutItemDescriptionToggle( $container );
 
+		// Bind loader visibility.
+		obj.bindLoaderEvents();
+
 		$document.trigger( 'afterSetup.tribeTicketsCommerceCheckout', [ $container ] );
 	};
 
+	/**
+	 * Bind loader events.
+	 *
+	 * @since TBD
+	 */
+	obj.bindLoaderEvents = function () {
+		$document.on( obj.customEvents.showLoader, obj.loaderShow );
+		$document.on( obj.customEvents.hideLoader, obj.loaderHide );
+	}
+
+	/**
+	 * Show the loader/spinner.
+	 *
+	 * @since TBD
+	 */
+	obj.loaderShow = function() {
+		tribe.tickets.loader.show( $( obj.selectors.checkoutContainer ) );
+	};
+
+	/**
+	 * Hide the loader/spinner.
+	 *
+	 * @since TBD
+	 */
+	obj.loaderHide = function() {
+		tribe.tickets.loader.hide( $( obj.selectors.checkoutContainer ) );
+	};
```

Comment:
let's move these above so that we define first and use them after. `bindLoaderEvents` uses `loaderShow` and `loaderHide`, so those should be defined first, and `bindCheckoutEvents` uses `bindLoaderEvents`, so we should move `bindLoaderEvents` above.